web/apps/labelstudio/src/components/HeidiTips/liveContent.json

```diff
@@ -76,20 +75,19 @@
         "url": "https://docs.humansignal.com/guide/auth_setup",
         "params": {
           "experiment": "organization_page_tip",
-          "treatment": "enable_sso"
+          "treatment": "enable_sso_live"
         }
       }
     },
     {
-      "title": "Want to automate task distribution?",
-      "content": "Create rules, automate how tasks are distributed to annotators, and only show tasks assigned to each annotator in their view.and control task visibility for each annotator.",
-      "closable": true,
+      "title": "Did you know?",
+      "description": "Label Studio now has a Starter Cloud offering optimized for small teams and projects.",
```

Comment:
Is it intentional that we repeat this twice? see line 18

Reply:
Actually, I see it three times again a few lines below.

Reply:
Almost -- it's so it shows up on 3 different pages: Auth, Projects and Organization. But... I like that idea too :)